src/views/Products/Products.test.jsx

```diff
@@ -0,0 +1,67 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import { AppWrapper } from '../../utils/testing';
+import Products from './';
+import categories from '../../mocks/en-us/product-categories.json';
+import products from '../../mocks/en-us/products.json';
+
+const renderComponent = () =>
+  render(
+    <AppWrapper>
+      <Products />
+    </AppWrapper>
+  );
+
+describe('Product Category Sidebar ', () => {
+  beforeEach(() => {
+    renderComponent();
+  });
+
+  it('renders data from API', async () => {
+    const categoryTextList = await screen.findAllByTestId('category-checkbox-text');
+    expect(categoryTextList[0].textContent).toBe(categories.results[0].data.name);
```

Comment:
Let's check them all 
```suggestion
        categoryTextList.forEach((category, index) => expect(category).toHaveTextContent(categories.results[index].data.name));

```

Reply:
Sure, thanks